yarn.lock

```diff
@@ -7236,12 +7236,12 @@ __metadata:
   linkType: hard
 
 "follow-redirects@npm:^1.0.0":
```

Comment:
`follow-redirects` is a dependency of `http-proxy`, which is a dependency of `http-proxy-middleware`, which is needed by `webpack-dev-server`, which is part of `react-scripts`. I could try upgrading `react-scripts` from 4.0.3 to 5.0.0 if we want.

Reply:
I tried that upgrade, and as one would expect, it introduces tons of dependency changes. Also 5.0.0 was released about a month ago, so it won't necessarily update follow-redirects (security fix is more recent).

Reply:
Webpack dev server is only used for local dev, so we should really dismiss this alert, since it's not exposed, rather than messing with it. That said, the way I normally handle this sort of thing is to delete the entry entirely and see what version `yarn install` puts when we run it.

Reply:
(and then run `yarn dedupe`)

Reply:
Thanks, I took that approach in my most recent commit:

1. Delete entries in yarn.lock that reference follow-redirects
2. yarn install
3. yarn dedupe

Reply:
Would it be possible to add documentation around these steps to [Security and Maintenance](https://github.com/DataBiosphere/terra-ui/wiki/Security-and-Maintenance)?

Reply:
@kyuksel definitely! That would have helped me immensely yesterday as I floundered around trying to figure out what to do.

Reply:
Updated https://github.com/DataBiosphere/terra-ui/wiki/Security-and-Maintenance#responsibilities